spec/support/factory_bot.rb

```diff
@@ -12,8 +12,10 @@
       factory.name =~ /_holdings?$/ || factory.name =~ /_searchworks_item$/ || factory.name =~ /^symphony_/
     end
 
-    FactoryBot.lint factories_to_lint unless config.files_to_run.one?
-  ensure
-    DatabaseCleaner.clean
+    begin
+      FactoryBot.lint factories_to_lint unless config.files_to_run.one?
```

Comment:
This effectively makes our factory linting completely useless by wrapping in a `begin`.

Reply:
Why do I get `SyntaxError:
  /Users/jgreben/Projects/sul-requests/spec/support/factory_bot.rb:16: syntax error, unexpected keyword_ensure, expecting keyword_end` when running rspec with the original file?

Reply:
Thanks, this passes on travis. My local system needs updating to catch up with this project.